kernel/syscalls/clone3.rs

```diff
@@ -0,0 +1,139 @@
+use crate::prelude::*;
+use crate::syscalls::SyscallHandler;
+use crate::{
+    ctypes::*
+};
+use kerla_runtime::address::UserVAddr;
+use bitflags::bitflags;
+
+
+bitflags! {
+    /// Flags used by [`clone()`] system call
+    /// Source: `/usr/include/linux/sched.h`
```

Comment:
Please avoid copying from the Linux source code even for a definition like this. Please refer musl instead.